pkgs/applications/emulators/fs-uae/launcher.nix

```diff
@@ -1,9 +1,4 @@
-{ lib
-, stdenv
-, fetchurl
-, gettext
-, makeWrapper
-, python3
+{ lib, stdenv, fetchurl, gettext, python3, wrapQtAppsHook, wrapGAppsHook, fsuae
```

Comment:
I think these formatting changes should be reverted, the style used prior to this PR makes diffs a lot clearer to read. Same goes for the other places were spacing was removed.

Reply:
Sorry about that, I thought nixpkgs moved to nixfmt already

Reply:
There are many discussions about nixfmt itself running around.

Especially, I use one attribute per line because it makes it clear what was changed in the diffs.
I don't need to read a long long line to look that you removed makeWrapper and is now using qt wrappers.